base.css

```diff
@@ -122,4 +122,13 @@
   .background-position-2 {
     background-position: right -10px bottom -80px;
   }
+
+  .background-fleet {
+    background-image: url("/truck.svg");
+    background-size: 94.87px 75.38px;
+  }
```

Comment:
I think this could be done inline as well with something like `bg-[size:94.87px_75.38px] bg-[url('truck.svg')]`